tests/dummy/app/pods/checkbox/template.hbs

```diff
@@ -1,172 +1,154 @@
-<div class="section">
-  <div class="title">frost-checkbox</div>
-  <div class="example">
-    <div class="title">Basic</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
-{{frost-checkbox id='basic'}}
-```"
-}}
+<div class="dummy-notification-container">
+{{#each notifications as |notification|}}
+  {{notification-message notification=notification}}
+{{/each}}
+</div>
+
+<div class='dummy-body text-input field'>
+  <div class='section'>
+
+    <div class='example'>
+      <div class='title'>Basic</div>
+      <div class='demo'>
+        {{frost-checkbox}}
+      </div>
+      <div class='snippet'>
+        {{format-markdown
+"```handlebars
+{{frost-checkbox}}
+```"}}
+      </div>
     </div>
-    <div class="demo">
-      {{frost-checkbox id="basic"}}
+
+   <div class='example'>
+      <div class='title'>Size</div>
+      <div class='demo'>
+        {{frost-checkbox size='large'}}
```

Comment:
I had that and then I removed it...  I figured it was just an example and the available options are in the readme.